src/include/parser/parsenodes.h

```diff
@@ -0,0 +1,249 @@
+#include "nodes.h"
+#include "pg_list.h"
+
+typedef enum SetOperation
+{
```

Comment:
I could not find any format.py under peloton folder, are there any written guides?